src/methods/install-app.js

```diff
@@ -67,7 +67,13 @@ export default (self, appId, redirect = false) => {
           result: resp.data
         }
       })
-  })
+  }
+
+  if (appBody && appBody.app_id) {
+    install(appBody)
```

Comment:
não deveria ter um return aqui também?

```suggestion
    return install(appBody)
```